script/DeployToBase.s.sol

```diff
@@ -69,72 +77,95 @@ contract DeployEthMultiVault is Script {
+            atomWalletInitialDepositAmount: 0.000003 ether, // Fee charged for purchasing vault shares for the atom wallet upon creation
+            atomCreationProtocolFee: 0.000003 ether // Fee charged for creating an atom
         });
 
         IEthMultiVault.TripleConfig memory tripleConfig = IEthMultiVault.TripleConfig({
-            tripleCreationProtocolFee: 0.0003 ether, // Fee for creating a triple
-            totalAtomDepositsOnTripleCreation: 0.00003 ether, // Static fee going towards increasing the amount of assets in the underlying atom vaults
-            atomDepositFractionForTriple: 900 // Fee for equity in atoms when creating a triple
+            tripleCreationProtocolFee: 0.000003 ether, // Fee for creating a triple
+            atomDepositFractionOnTripleCreation: 0.000003 ether, // Static fee going towards increasing the amount of assets in the underlying atom vaults
+            atomDepositFractionForTriple: 300 // Fee for equity in atoms when creating a triple
         });
 
         IEthMultiVault.WalletConfig memory walletConfig = IEthMultiVault.WalletConfig({
-            permit2: IPermit2(address(permit2)), // Permit2 on Base
-            entryPoint: entryPoint, // EntryPoint address on Base
+            permit2: IPermit2(address(permit2)), // Permit2 on Linea
+            entryPoint: entryPoint, // EntryPoint address on Linea
             atomWarden: atomWarden, // atomWarden address
             atomWalletBeacon: address(atomWalletBeacon) // Address of the AtomWalletBeacon contract
         });
 
         IEthMultiVault.VaultFees memory vaultFees = IEthMultiVault.VaultFees({
             entryFee: 500, // Entry fee for vault 0
             exitFee: 500, // Exit fee for vault 0
-            protocolFee: 250 // Protocol fee for vault 0
+            protocolFee: 100 // Protocol fee for vault 0
+        });
+
+        IEthMultiVault.BondingCurveConfig memory bondingCurveConfig = IEthMultiVault.BondingCurveConfig({
+            registry: address(bondingCurveRegistry),
+            defaultCurveId: 1 // Unused in this edition of EthMultiVault
         });
 
         // ------------------------------- Bonding Curves----------------------------------------
 
         // Deploy BondingCurveRegistry and take temporary ownership to add the curves
-        bondingCurveRegistry = new BondingCurveRegistry(msg.sender);
-        console.logString("deployed BondingCurveRegistry.");
+        bondingCurveRegistry = new BondingCurveRegistry();
+        console.logString("deployed BondingCurveRegistry implementation.");
+
+        address bondingCurveRegistryImplementation = address(bondingCurveRegistry);
+
+        bytes memory bondingCurveRegistryInitData =
+            abi.encodeWithSelector(BondingCurveRegistry.initialize.selector, msg.sender);
+
+        // Deploy BondingCurveRegistry proxy
+        bondingCurveRegistryProxy = new TransparentUpgradeableProxy(
```

Comment:
I don't think the bonding curve registry is a proxy, and we don't need this.  It doesn't store any important state (outside of the bonding curve IDs), so if we ever wanted a new one we can just deploy a new one and point the multivault config at it.

If we did this, we would need to ensure that any new registry just has the same bonding curve IDs as the old one.  Because the curve IDs are tied to the economic state in the MultiVault, we never want to change them.  Only add to them.